make_queue/models/models.py

```diff
@@ -398,61 +400,61 @@ def save(self, **kwargs):
             raise ValidationError("Rule is not valid")
         return super().save(**kwargs)
 
-    @staticmethod
-    def valid_time(start_time, end_time, machine_type):
+    @classmethod
+    def valid_time(cls, start_time: datetime, end_time: datetime, machine_type: MachineType) -> bool:
         """
         Checks if a reservation in the supplied period is allowed by the rules for the machine type.
 
-        :param start_time: The start time (datetime) of the reservation
-        :param end_time: The end time (datetime) of the reservation
+        :param start_time: The start time of the reservation
+        :param end_time: The end time of the reservation
         :param machine_type: The type of machine for the reservation
         :return: A boolean indicating if the reservation follows the rules
         """
```

Comment:
Same as the other comment, please add cls to the documentation.

Reply:
See my reply to the other comment ✨